src/interfaces/utils.js

```diff
@@ -25,14 +25,33 @@
 * for the JavaScript code in this file.
 *
 */
+const permissions = {
+	profiles: {
+		createOrUpdate: ['system'],
+		read: ['system', 'importer', 'transformer'],
+		query: ['system'],
+		remove: ['system']
+	},
+	blobs: {
+		query: ['all'],
+		read: ['all'],
+		create: ['system', 'creator'],
+		update: ['system', 'importer', 'transformer'],
+		abort: ['all'],
+		remove: ['system'],
+		removeContent: ['system'],
+		readContent: ['all']
+	}
+};
 
-export function hasPermission(profile, user) {
-	const permitted = profile.auth.groups.some(profileGroup => {
-		return user.groups.some(userGroup => userGroup === profileGroup || userGroup === 'admin');
-	});
-	return permitted;
-}
+export function hasPermission(type, command, profile, bgroups = []) {
```

Comment:
Change parameter names to be more clear:

- *profile* -> **userGroups**
- *bgroups* -> **permittedGroups**

Also, it's unecessary to pass the whole profile object here. Passing `profile.auth.groups` is sufficient.